shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command.report;
+
+import alluxio.client.block.RetryHandlingBlockMasterClient;
+import alluxio.client.MetaMasterClient;
+import alluxio.client.RetryHandlingMetaMasterClient;
+import alluxio.master.MasterClientConfig;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.wire.BlockMasterInfo;
+import alluxio.wire.MasterInfo;
+
+import java.util.Map;
+
+/**
+ * Print Alluxio cluster summarized information.
+ */
+public class SummaryCommand {
+  /** Print Alluxio cluster summarized information.*/
+  public static void printSummary() {
```

Comment:
could we print in JSON format to make it easier to use the tool programmatically? It would also solve challenges with keeping the indentation consistent

Reply:
JSON is not pretty printing out. Is GSON(com.google.code.gson) a good choice?
GSON will help to print JSON pretty. 

Personally, I prefer clean output without JSON format when I use CLI.